src/light-signup.scss

```diff
@@ -59,25 +58,27 @@
   margin-bottom: 5px;
 }
 
-.n-light-signup__no-spam {
+.o-light-signup__no-spam {
   @include oTypographySans(s);
   margin: 0;
   display: inline-block;
 }
 
-.n-light-signup__close {
-  @include nextIcon(
-	$icon-name: cross_disk,
-	$size: 20,
-	$color: #4A4A4A
+.o-light-signup__close {
+  @include oFtIconsGetSvg(
```

Comment:
Probably worth moving to o-icons directly https://github.com/Financial-Times/o-icons/pull/2 and putting the cross-disk icon there. Do you have the icon?